packages/lms/src/routes/+layout.server.ts

```diff
@@ -1,8 +1,16 @@
 import { getEntityMetaTree } from '$lib/utils/getEntityMetaTree';
 import type { LayoutServerLoad } from './$types';
+import path from 'path';
 
-export const load: LayoutServerLoad = () => {
-  return {
-    sitemap: getEntityMetaTree('src/routes/content')
-  };
+export const load: LayoutServerLoad = ({ url }) => {
+
+  try {
+    const navPath = path.join('src/routes', path.join(...url.pathname.split('/').splice(1, 3)))
```

Comment:
This fixes the problem with no navigation showing, its due to the replacement of " " with "%20" in the url.
```suggestion
    const navPath = path.join('src/routes', path.join(...decodeURIComponent(url.pathname).split('/').splice(1, 3)));
```